tests/aggregation_tests.py

```diff
@@ -11,278 +11,282 @@
 
-        # WHEN grouped entries get created
-        main.create_count()
+        main.aggregate()
 
-        # THEN count table should have 2 rows corresponding to different risks, with properly grouped entries
+        #count table should have 2 rows corresponding to different risks, with properly grouped entries
         self.cursor.execute('select * from count;')
         self.assertEqual(
             self.cursor.fetchall(),
             [
-
-                # Risk type 99 - one entry from hostB
-                (99, 'US', 12252L, '2016-09-26', 'monthly', 1),
-
-                # Risk type 2 - 2 total count: one entry from hostB, and two from hostA, which grouped into one
-                (2, 'US', 12252L, '2016-09-26', 'monthly', 2)
+                (datetime.datetime(2016, 9, 29, 0, 0), 1, 'US', 12252L, 1, 0.0),  
+                (datetime.datetime(2016, 9, 29, 0, 0), 2, 'US', 12252L, 2, 0.0)
             ])
 
+
     def test_group_by_country(self):
-        # GIVEN 3 entries of the same risk and week, two of which are from one country, but different asn
-        ntp_scan_csv = dedent('''\
+        # GIVEN 3 entries of the same risk and day, two of which are from one country
+        scan_csv = dedent('''\
         ts,ip,risk_id,asn,cc
-        2016-09-28T00:00:01+00:00,190.81.134.82,2,4444,US
+        2016-09-29T00:00:01+00:00,190.81.134.82,2,12252,US
         2016-09-29T00:00:01+00:00,190.81.134.11,2,12252,US
-        2016-09-29T00:00:01+00:00,190.81.134.11,2,3333,DE
+        2016-09-29T00:00:01+00:00,190.81.134.34,2,3333,DE
         ''')
-        self.cursor.copy_expert("COPY logentry from STDIN csv header", StringIO(ntp_scan_csv))
+        self.cursor.copy_expert("COPY logentry from STDIN csv header", StringIO(scan_csv))
 
-        # WHEN grouped entries get created
-        main.create_count()
-        main.create_count_by_country()
+        main.aggregate()
 
-        # THEN count_by_country table should have 2 entries which get grouped, and one entry which stands alone
-        self.cursor.execute('select * from count_by_country;')
+        # count table should have 2 entries - US with count of 2, and DE with count of 1
+        self.cursor.execute('select * from count;')
         self.assertEqual(
             self.cursor.fetchall(),
             [
-                (2, 'DE', '2016-09-26', 1L, 0.0, 0),
-                (2, 'US', '2016-09-26', 2L, 0.0, 0)   # 2 entries grouped by country
+                (datetime.datetime(2016, 9, 29, 0, 0), 2, 'DE', 3333L, 1, 0.0),
+                (datetime.datetime(2016, 9, 29, 0, 0), 2, 'US', 12252L, 2, 0.0)  
             ])
 
-    def test_group_by_risk(self):
-        # GIVEN 3 entries, of the same week, two of which have same risk type, but different countries
-        ntp_scan_csv = dedent('''\
+
+    def test_group_by_asn(self):
+        # GIVEN 3 entries of the same risk, country and day, but with different asn
```

Comment:
Should say "# GIVEN 3 entries of the same risk, country and day, two of which have the same asn"